### PR TITLE
Wrap functions in function_exists

### PR DESCRIPTION
**Pull request by olivernybroe**
Wrap all functions which are not namespaced in function_exists.

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]

File: src/Functions.php [hunk 1]
```diff
@@ -12,95 +12,111 @@
...
- * a closure that contains the test expectations.
- *
- * @return TestCall|TestCase|mixed
- */
-function it(string $description, Closure $closure = null): TestCall
-{
-    $description = sprintf('it %s', $description);
-
-    return test($description, $closure);
+if (!function_exists('')) {
```
Comment (tomasnorre) [inline src/Functions.php:84]: Isn't that a mistake that the '' is empty? Or am I reading something wrong?

Comment (olivernybroe) [reply src/Functions.php]: haha, thanks a lot! Great catch. 

I made a quick autohotkey for this, guess I missclicked here 😆

**Comment by nunomaduro** [comment]
@owenvoke Can you merge and tag?

**Comment by nunomaduro** [comment]
@olivernybroe Meant you lol

**Comment by nunomaduro** [comment]
Thank you.

**Pull request merged**